mobile/src/components/course/course.tsx

```diff
@@ -48,17 +74,31 @@ const Course: FC = () => {
     return <Spinner isOverflow />;
   }
 
+  if (!course) {
+    return <Text>There is no course with provided id </Text>;
+  }
+
   return (
     <ScrollView>
       <View style={styles.container}>
-        <Text style={styles.h1}>{dataCourse?.title}</Text>
+        <Text style={styles.h1}>{course?.title}</Text>
+        <View style={styles.currentCategory}>
+          <Category
+            keyName={currentCategory?.key ?? 'unknown'}
+            name={currentCategory?.name ?? 'Unknown'}
+            isActive={false}
+          />
+        </View>
+
         <Image
           style={styles.image}
-          source={{ uri: course?.imageUrl ?? getImageUri(defaultCourseImage) }}
+          source={{
+            uri: course?.imageUrl ?? getImageUri(defaultCourseImage),
+          }}
         />
         <Text style={styles.h2}>About this course</Text>
-        {Boolean(course?.description) && (
-          <Content html={dataCourse?.description} width={width} />
+        {course.description && (
```

Comment:
It's important to convert strings to boolean in conditional rendering especially in react-native, because if we have empty string it will cause error
```
const hasDescription = Boolean(course.description);
//
{hasDescription && (
```